### DISCUSSION TITLE
Password import certificate

### DISCUSSION THREAD

**Discussion by lutskboy** [mitmproxy/mitmproxy#5925] [Q&A] [answered]
Hi
i am trying to import the certificate
but asks for a password
what password should i enter
https://www.youtube.com/watch?v=RI_1SSNCdp8

**Comment by mhils** [comment]
Are you using the latest mitmproxy release? There is no password on the certificate.

**Reply by lutskboy** [reply]
> Are you using the latest mitmproxy release? There is no password on the certificate.

yes. latest version mitm. windows 8.1

**Reply by mhils** [reply]
This might be a Windows 8.1 problem then, which is not supported anymore.

**Reply by lutskboy** [reply]
> Тогда это может быть проблема Windows 8.1, которая больше не поддерживается.

v8 works well on windows 8.1 on my other computer. but on laptop not work v8 v9

**Comment by lutskboy** [comment, marked as answer]
my solution. download .pem and convert to .der
fine work on Windows 8.1 v8 and v9 mitmproxy
